Makefile

```diff
@@ -17,42 +15,36 @@ dev:
 livereload:
 	$(MANAGE) livereload $(IP):$(PORT)
 
-.PHONY: venv
-venv:
-	python3 -m venv $(VENV)
-	@echo "When developing, activate the virtualenv with 'source .venv/bin/activate' so Python can access the installed dependencies."
+.PHONY: conda-scratch
+conda-scratch:
+	# Makes hknweb environment from scratch (overwriting old one if exists)
+	conda create -n $(CONDA_ENV) python=$(PYTHON_VERSION) -y
+	@echo "When developing, activate the HKNWeb Conda environment with 'conda activate $(CONDA_ENV)' so Python can access the installed dependencies."
+
+.PHONY: conda
+conda:
+	@# We don't use "-y" because if environment recrated, it will destroy and reinstall ... since it is an all "yes"
+	@#  Just confirm to the default options (option given is either: Yes install OR No remove and reinstall)
+	echo -e "\n" | conda create -n $(CONDA_ENV) python=$(PYTHON_VERSION)
```

Comment:
Don't need this -e here.

Reply:
Needed to confirm install on first time, but if the environment exists, it will destroy the old one and recreate

The default option respectively is Yes Install, but No Reinstall

"-e" also allows for the use of \n as newline instead of it as separate characters (like "\" and "n")

Reply:
What about `if [ conda env list | grep $CONDA_ENV | $(wc -l) -lt 1 ] then conda create ... fi`? We can check if the env by that name already exists

Reply:
Conda Create checks if exists already, and we just use the default option of **deny** the overwrite with a simple new line

While the solution proposed is the alternate and does the similar behavior, next level consideration imo is what's simpler for future CompServ to read, and the current line is easier imo

Reply:
AgreeGe, makes sense